keda/values.yaml

```diff
@@ -226,8 +226,7 @@ securityContext:
       drop:
       - ALL
     allowPrivilegeEscalation: false
-    ## Metrics server needs to write the self-signed cert. See FAQ for discussion of options.
-    # readOnlyRootFilesystem: true
+    readOnlyRootFilesystem: true
```

Comment:
Is cert manager required feature or not? If not, LGTM

Reply:
No it isn't. With or without cert manager, the cert is a dep that will be always mounted in the metrics server (generated by cert-manager or by keda-operator)